### PR TITLE
Upgrade webrtc lib in licode

### PR DESCRIPTION
**Pull request by j4y**

### CODE REVIEW

File: erizo/src/erizo/WebRtcConnection.cpp [hunk 1]
```diff
@@ -580,7 +583,8 @@ namespace erizo {
       p_.comp = comp;
 //      p_.type = (transport->mediaType == VIDEO_TYPE) ? VIDEO_PACKET : AUDIO_PACKET;
       p_.type = type;
-      p_.length = length;
+      // TODO: modify dataPacket to use size_t
```
Comment (MorikoHandford) [inline erizo/src/erizo/WebRtcConnection.cpp]: When is this change to modify dataPacket going to be done?

File: erizo/src/erizo/WebRtcConnection.h [hunk 2]
```diff
@@ -11,7 +11,9 @@
 #include "MediaDefinitions.h"
 #include "Transport.h"
 #include "Stats.h"
-#include "rtp/webrtc/fec_receiver_impl.h"
+// #include "rtp/webrtc/fec_receiver_impl.h"
```
Comment (MorikoHandford) [inline erizo/src/erizo/WebRtcConnection.h]: Can this line just be removed?  We have it in the git history.

File: erizo/src/erizo/media/ExternalOutput.h [hunk 3]
```diff
@@ -3,7 +3,9 @@
 
 #include "../MediaDefinitions.h"
 #include "rtp/RtpPacketQueue.h"
-#include "rtp/webrtc/fec_receiver_impl.h"
+//#include "rtp/webrtc/fec_receiver_impl.h"
```
Comment (MorikoHandford) [inline erizo/src/erizo/media/ExternalOutput.h]: ditto removing this (and any further commented code).

**Comment by MorikoHandford** [comment]
Except for the commented code, looks good to me.  :shipit:

**Comment by j4y** [comment]
I'll send a new PR with the updated changes.

**Pull request closed without merging**